.env.example

```diff
@@ -10,6 +10,7 @@ AWS_SECRET_ACCESS_KEY=
 AWS_S3_BUCKET_URL=
 GITHUB_CLIENT_ID=
 GITHUB_CLIENT_SECRET=
+GITHUB_USER_OAUTH_URL=
```

Comment:
Thanks a lot for this tagging @sofiarm21 @Ovifer13 as a reminder to get in the habit of making sure to add environment variables in the example file I often forget to do this and it may result in failed deployments if we forget that a new environment variable is needed when going to production